taichi/program/matrix.h

```diff
@@ -0,0 +1,36 @@
+#pragma once
+
+#include "taichi/ir/type.h"
+
+namespace taichi::lang {
+
+/**
+ * Matrix wrapper used in AOT.
+ */
+class Matrix {
+ public:
+  explicit Matrix(const uint32_t &length,
+                  const DataType dtype,
+                  const intptr_t &data)
+      : length_(length), dtype_(dtype), data_(data) {
+  }
+
+  DataType dtype() const {
+    return dtype_;
+  }
+
+  uint32_t length() const {
```

Comment:
nit: better indicate whether the length is in bytes or # of elements

Reply:
Solved with a comment.